mozilla_aws_cli/static/index.js

```diff
@@ -107,6 +107,9 @@ const pollState = setInterval(async () => {
         // show the roles
         const roles = await response.json();
         showRoles(roles);
+    } else if (remoteState.state === "restart_auth") {
+        setMessage("Redirecting to identity provider...");
+        window.location.replace(remoteState.value.idpUrl);
```

Comment:
You should probably do this as `document.location = remoteState.value.idpUrl`, if it is in fact redirecting?

Reply:
Reading [this](https://stackoverflow.com/a/506004/168874) it sounded like using `replace` acts most like a redirect and prevents the user from having negative back button experience. What happens with `document.location` if the user clicks the back button? Does it skip over the page initiating the redirect and go the page before the page? Or does it go back to the redirect page which then redirects them forward again?

Reply:
Ahh, you're probably right. Maybe replace the other redirect in `index.js` so that they match?

Reply:
Good idea, I've changed the other instance of the redirect to match this in 8cf7c507b3979198a8c753b02ef0928b211d7914